package.json

```diff
@@ -28,7 +28,9 @@
     "deploy": "git-directory-deploy --directory build/ --branch ${DEPLOY_BRANCH:-build} --repo=${DEPLOY_REPOSITORY:-https://$GITHUB_TOKEN@github.com/konnectors/cozy-konnector-template.git}"
   },
   "dependencies": {
-    "cozy-konnector-libs": "3.8.1"
+    "cozy-konnector-libs": "3.8.1",
```

Comment:
This version of cozy-konnector-libs is a little outdated (4.1.4 now), same for cozy-jobs cli

Reply:
Done